react/features/filmstrip/components/native/TileView.js

```diff
@@ -0,0 +1,324 @@
+// @flow
+
+import React, { Component } from 'react';
+import {
+    ScrollView,
+    TouchableWithoutFeedback,
+    View
+} from 'react-native';
+import { connect } from 'react-redux';
+
+import {
+    getNearestReceiverVideoQualityLevel,
+    setMaxReceiverVideoQuality
+} from '../../../base/conference';
+import {
+    DimensionsDetector,
+    isNarrowAspectRatio,
+    makeAspectRatioAware
+} from '../../../base/responsive-ui';
+
+import Thumbnail from './Thumbnail';
```

Comment:
Please put in alphabetical order

Reply:
You're talking about `./Thumbnail` and `./styles`? I think this is the desired order. At some point it was decided capitals should come before lowercase.

Reply:
Grr... I hate these unwritten rules. My understanding is that capitals come first only if they are the same letter. E.g. 'T' comes before 't', but won't come before 's'. But I may be wrong, so just ignore it, I'm fine this way too